docs/administration/spaces/index.md

```diff
@@ -4,76 +4,134 @@ description: Spaces setup and configuration.
+Spaces let you partition your Octopus Server so that different teams can only access the projects, environments, and infrastructure they work with from the Spaces they are members of.
 
-Using Spaces keeps the different projects and infrastructure your teams use completely separate, which means something configured in **SpaceA**, is not available to projects in **SpaceB**. This makes it easier for large organizations with multiple teams using Octopus because each team member will only see the projects, environments, and infrastructure that is available in their space.
+Using spaces keeps the different projects and infrastructure your teams use completely separate, which means something configured in **SpaceA**, is not available to projects in **SpaceB**. This makes it easier for large organizations with multiple teams using Octopus because each team member will only see the projects, environments, and infrastructure that is available in their space.
 
-By default, every instance of Octopus Server since **Octopus 2018.11** comes with a default space, however, if you are not planning to use multiple Spaces, this default Space exists in the background and you do not to configure it or manage it. For organizations not using Spaces the default space can be safely ignored.
+By default, every instance of Octopus Server since **Octopus 2018.11** comes with a default space. However, if your organization is not planning to use multiple spaces, this default Space can be safely ignored and doesn't require configuration or management. 
 
 ## Managing Spaces
 
 Spaces are managed by navigating to {{Configuration,Spaces}}.
 
-### The Space Owner
+An Octopus Administrator, or a team member with sufficient permission, is able to create, remove or modify spaces from this screen. It is also possible to [nominate a default space](#change-the-default-space), or [disable the default space entirely](#disable-the-default-space). Each space has a logo, which is also shown in the [space switcher](#switching-between-spaces) to make it easy to identify which space is currently focused upon by the UI. There is also a search filter to quickly find the spaces that you are interested in managing, this searches over the name and description fields of each space.
 
-Each Space has a Space Owner. The Space Owner is the Admin for that Space and is responsible for adding team members to the Space and assigning permissions to the members of the space.
+![Spaces configuration page](spaces-configuration.png)
 
-### Create a Space
+### The Space Manager
+
+Each space has a **Space Manager**. The space manager is the Administrator for that Space and is responsible for managing users and teams to the space and assigning permissions to them.
+
+When creating a new space, you are required to nominate a team member (or a team) to the role of space manager. This space manager is then responsible for [managing teams and permissions](/docs/managing-users-and-teams) within that space.
```

Comment:
This paragraph can probably be combined with the previous one - mostly says the same thing.